src/yuzu/configuration/config.cpp

```diff
@@ -876,6 +897,14 @@ void Config::SavePlayerValues() {
                          QString::fromStdString(player.buttons[i]),
                          QString::fromStdString(default_param));
         }
+        for (int i = 0; i < Settings::NativeMotion::NumMotions; ++i) {
+            const std::string default_param =
+                InputCommon::GenerateKeyboardParam(default_motions[i]);
+            WriteSetting(QStringLiteral("player_%1_").arg(p) +
```

Comment:
```suggestion
            WriteSetting(QStringLiteral("player_%1_%2").arg(p, QString::fromStdString(Settings::NativeMotion::mapping[i]))
```

This way the entire string is formatted in place in a single pass. Using operator+ here would churn strings, since it needs to discard a temporary before creating the final string.